server/src/controllers/activateAccount.js

```diff
@@ -9,8 +9,7 @@ const activateSchema = Joi.object({
 });
 
 module.exports = (req, res, next) => {
-  const active = req.body.active === 'true';
-  const { admin } = req.body;
+  const { admin, active } = req.body;
```

Comment:
I think lina made this like this for some reason @LinaYahya

Reply:
@Amoodaa  @omar-dahdouh It works in our case without this conversion, but if we need to check if active then admin, we need this line